cloudinit/sources/DataSourceOVF.py

```diff
@@ -140,6 +141,10 @@ def _get_data(self):
             try:
                 cf = ConfigFile(vmwareImcConfigFilePath)
                 self._vmware_cust_conf = Config(cf)
+                post_gc_status = self._vmware_cust_conf.post_gc_status
+                if post_gc_status:
+                    set_gc_status("Started")
```

Comment:
Still looking to have the fix requested in https://github.com/canonical/cloud-init/pull/215#discussion_r393902593 fixed here.

Reply:
Hi OddBloke,
Sorry I misunderstood that you agreed to not move this check into set_gc_status() function. If I still need to move it, then this function will looks like "set_gc_status(gcMsg, vmware_cust_conf)". Each time we set the gc status, we need to parse the vmware cust conf and read boolean "MISC|POST-GC-STATUS". It might be not efficient, do you agree? Thanks.

Reply:
> Each time we set the gc status, we need to parse the vmware cust conf and read boolean "MISC|POST-GC-STATUS". It might be not efficient, do you agree?

I may not be following the logic through correctly, but it looks to me like `ConfigFile` takes care of reading the config file a single time, so I don't believe this is an accurate description of the additional work that would be performed. I think we're really just talking about a few additional Python dict operations, which is not a significant cost.

Reply:
Address your comments, PTAL, thanks a lot.